Packs/Traceable/Integrations/Traceable/Traceable.yml

```diff
@@ -99,11 +100,56 @@ configuration:
   - TOR Exit Node
   - BOT
   - Unknown
+- display: Traceable Platform Endpoint URL
+  defaultvalue: https://app.traceable.ai
+  name: app_url
+  required: false
+  type: 0
 - display: Incident type
   defaultvalue: Exploit
   name: incidentType
   required: false
   type: 13
+- display: Ignore Status Codes
+  name: ignoreStatusCodes
+  defaultvalue: 400-499
+  type: 0
+  required: false
+- display: Incident optional field list
+  name: optionalDomainEventFieldList
+  defaultvalue: actorDevice,actorEntityId,actorId,actorScoreCategory,actorSession,anomalousAttribute,apiName,apiUri,category,ipAbuseVelocity,ipReputationLevel,securityEventType,securityScore,serviceId,serviceName,actorScore,threatCategory,type
```

Comment:
Do you need this (defaultvalue) if you have the list of options?

Reply:
Hi @ShirleyDenkberg the idea was to have it fetch as many fields to start with, and the customer can take a conscious decision to remove or retain the ones that they wish.